src/zenml/logging/__init__.py

```diff
@@ -18,7 +18,7 @@
 STEP_LOGS_STORAGE_INTERVAL_SECONDS: int = 15
 
 # How many messages to buffer before uploading logs to the artifact store
-STEP_LOGS_STORAGE_MAX_MESSAGES: int = 100
+STEP_LOGS_STORAGE_MAX_MESSAGES: int = 1000
```

Comment:
In constants you define `1000` again as the default when reading the env variable, but somehow don't use this one?

Reply:
This is only defined in the `logging/__init__.py`. In the constants, we have the `LOGS_STORAGE_MAX_QUEUE_SIZE` coming for the env variable. Maybe that created the confusion?

Reply:
Ah yes that confused me, sorry! Just weird that they're in separate places IMO